pkg/kwok/server/debugging_logs.go

```diff
@@ -293,10 +293,7 @@ func readLogs(ctx context.Context, logsFile string, opts *logOptions, stdout, st
 
 				if recreated {
 					newF, err := os.Open(logsFile)
-					if err != nil {
-						if os.IsNotExist(err) {
-							continue
```

Comment:
https://github.com/kubernetes/kubernetes/blob/5ceb99dc6bae8a3013192439bdbee942d14de190/pkg/kubelet/kuberuntime/logs/logs.go#L366

You can try to apply this change upstream, not this